test/ui/registerTests.ts

```diff
@@ -0,0 +1,64 @@
+import webdriver from 'selenium-webdriver';
+import { expect } from 'chai';
+
+describe('register page', async () => {
+    const random = () => Math.random().toString(36).substring(7);
+    const validEmail = `${random()}@${random()}.com`;
+    const validPassword = `${random()}!1Aa`;
+
+    let driver: webdriver.WebDriver;
+
+    beforeEach(async () => {
+        driver = new webdriver.Builder().
+            withCapabilities(webdriver.Capabilities.chrome()).
+            build();
+    });
+    
+    afterEach(async () => {
+        await driver.quit();
+    });
+
+    it('should display the register form', async () => {
+        await driver.get(process.env.UI_TEST_URL + '/register');
+        const form = await driver.findElement(webdriver.By.css('form'));
+        expect(await form.isDisplayed()).to.be.true;
+    });
+
+    it('should submit the register form with valid input', async () => {
+        await driver.get(process.env.UI_TEST_URL + '/register');
+        const emailInput = await driver.findElement(webdriver.By.id('email'));
+        const passwordInput = await driver.findElement(webdriver.By.id('password'));
+        const registerButton = await driver.findElement(webdriver.By.css('button[type="submit"]'));
+
+        await emailInput.sendKeys(validEmail);
+        await passwordInput.sendKeys(validPassword);
+        await registerButton.click();
```

Comment:
It would be good to check you are redirected to login and see the success message.